<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="castle_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>297</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>297</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>367</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="next_castle">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="castle_0">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>331</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="map.qrc">:/map/Debug/map.png</pixmap>
    </property>
   </widget>
   <widget class="QTextBrowser" name="traveler_status">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="health_bar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>勇者状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>下一个可以前往的城堡:</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>castle_1</zorder>
   <zorder>castle_3</zorder>
   <zorder>castle_4</zorder>
   <zorder>castle_5</zorder>
   <zorder>castle_6</zorder>
   <zorder>castle_7</zorder>
   <zorder>castle_8</zorder>
   <zorder>castle_9</zorder>
   <zorder>castle_10</zorder>
   <zorder>castle_11</zorder>
   <zorder>castle_12</zorder>
   <zorder>castle_13</zorder>
   <zorder>castle_14</zorder>
   <zorder>castle_2</zorder>
   <zorder>next_castle</zorder>
   <zorder>castle_0</zorder>
   <zorder>traveler_status</zorder>
   <zorder>health_bar</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionFreePlay"/>
    <addaction name="actionMoreTreasure"/>
    <addaction name="actionMoreLife"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFreePlay">
   <property name="text">
    <string>自由游戏</string>
   </property>
  </action>
  <action name="actionMoreTreasure">
   <property name="text">
    <string>最多宝藏</string>
   </property>
  </action>
  <action name="actionMoreLife">
   <property name="text">
    <string>最多生命</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="map.qrc"/>
 </resources>
 <connections/>
</ui>
